Sources_v3/StreamChatUI/ChatChannel/MessageComposerSuggestionsCollectionView.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class MessageComposerSuggestionsCollectionView<ExtraData: ExtraDataTypes>: UICollectionView,
+    UIConfigProvider,
+    AppearanceSetting,
+    Customizable {
+    override open func didMoveToSuperview() {
+        super.didMoveToSuperview()
+        guard superview != nil else { return }
+
+        setUp()
+        (self as! Self).applyDefaultAppearance()
+        setUpAppearance()
+        setUpLayout()
+        updateContent()
+    }
+
+    // MARK: - Init
+
+    public required init(layout: UICollectionViewLayout) {
```

Comment:
This is required, collectionView always needs the layout for its correct init. I am open to any other option how we could replace this with something else :)

Reply:
I think this is correct. It's nice to let customers provide their own layout if needed.